HotTips/Tips/EDI_CutLine.md

```diff
@@ -0,0 +1 @@
+﻿Press **Shift+Del** (or **Ctrl + L**) to cut the line where the cursor is. That way it is on the clipboard where you can get it again.
```

Comment:
Shift +Del is "Cut". It will cut whatever is selected, or the whole line if nothing is selected.